src/Constraints/AdditionalProperties.php

```diff
@@ -31,7 +31,7 @@ public static function validate(
 
         if ($parameter === false) {
             return new ValidationError(
-                'Additional properties are not allowed.',
+                'Additional properties are not allowed: \'' . implode('\', \'', $diff) . '\'.',
```

Comment:
Nitpicking, but could you use a sprintf instead of string concatenation here so it's consistent with the rest of the constraints?  [Here's an example](https://github.com/thephpleague/json-guard/blob/master/src/Constraints/Type.php#L97).  Thanks!